README.md

```diff
@@ -21,6 +21,37 @@
 
 Native Rust implementation of [Apache Iceberg](https://iceberg.apache.org/).
 
+## RoadMap
+
+### Catalog
+
+| Catalog Type | Status      |
+|--------------|-------------|
+| Rest         | Done        |
+| Hive         | In Progress |
+| Sql          | Not Started |
+| Glue         | Not Started |
+| DynamoDB     | Not Started |
+
+### FileIO
+
+| FileIO Type | Status      |
+|-------------|-------------|
+| S3          | Done        |
+| Local File  | Done        |
+| GCS         | Not Started |
+| HDFS        | Not Started |
+
+### Table API
+
+* Table reader, including planning, reading into arrow record batch, etc.
```

Comment:
Maybe we should specific this a bit more. Planning also consists of partition evaluation, metrics evaluation, support for schema evolution, applying positional/equality deletes etc.